tests_e2e/test_browser.py

```diff
@@ -0,0 +1,163 @@
+from datetime import date
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import os
+from threading import Thread
+from todo_app import app
+from todo_app.board import Board
+
+CHROME_DRIVER_PATH = '/path/to/your/chromedriver'
+
+
+@pytest.fixture(scope='module')
+def driver():
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    with webdriver.Chrome(CHROME_DRIVER_PATH, options=opts) as driver:
```

Comment:
Nicer than asking each user to set their own chromedriver path (which we should do with an environment variable if we were to do it this way, otherwise pipelines will be painful!) would be to avoid specifying a path at all & just let the user add the executable to their PATH which will be searched by default

Reply:
Cheers, done